src/ucp/tag/eager_snd.c

```diff
@@ -293,7 +295,7 @@ static ucs_status_t ucp_tag_eager_generic_single(uct_pending_req_t *self)
         return status;
     }
 
-    ucp_tag_eager_generic_complere(self);
+    ucp_tag_eager_generic_complete(self, UCS_OK);
```

Comment:
yes, but this commit is only about the API, and this change was made only for the sake of a successful build. Next phase will be the implementation, where such issues will be addressed.